optimum/neuron/utils/optimization_utils.py

```diff
@@ -37,7 +37,7 @@ def _custom_badbmm(a, b):
         if self.upcast_softmax:
             attention_scores = attention_scores.float()
 
-        # attention_probs = attention_scores.softmax(dim=1).permute(0, 2, 1)
+        # attention_probs = attention_scores.softmax(dim=1).permute(0,2,1)
```

Comment:
To remove?

Reply:
this line was promised to replace the workaround below in 2.12 release, which is not yet the case, I would keep it and hopefully it will work in the next release